src/QMCDrivers/MCPopulation.cpp

```diff
@@ -85,7 +85,12 @@ void MCPopulation::allocateWalkerStuffInplace(int walker_index)
 
 void MCPopulation::createWalkers(IndexType num_walkers)
 {
+  num_walkers *= 2;
```

Comment:
What is this?

Reply:
We setup reserve walker elements to avoid dynamic allocation during the simulation.  Initially this was just to avoid some lifecycle issues that the unified design fell prey to by not relying on the default Walker copy constructor.  But looking forward to a native CUDA/HIP/ROCM implementation we also want 0 allocation during the run so prespawning a reserve makes sense.  For purposes of establishing a working batched implementation for moderate problem sizes I used a magic number.  Will fix.